### PR TITLE
Added bug handling/ triaging readme

### PR DESCRIPTION
**Pull request by abroddrick**
- Adds readme for bug handling

## Context for reviewers

summarized the bug handling steps we discussed in engineering huddle

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: docs/product/product_strategy.md [hunk 1]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+
+### Steps for dealing with Critical Bugs
+
+1. Once the critical bug ticket is refined and the bug is clear, an engineer should be assigned to work on it. (No ticket, no work)
+2. At the same point, two other engineers should be assigned to review the PR once it's made. One of the reviewing engineers can be subsititued for a designer if this is a design/content/other user facing bug fix.
+3. In the case where the engineering lead is not responsive/ unavailable to assign the ticket immediatley, other engineers should see the public slack message and volunteer as needed. The product team will be backup to make sure someone is assigned or volunteers for the ticket/PR review ASAP.
+4. Once done the developer, now needs to make a PR as usual but should tag the assigned PR reviewers in a public slack message stating that the PR is now waiting on their review. These reviewers should drop other tasks in order to review this promptly.
+5. See the Making bug fixes on stable during production readme for how to push changes to stable once the PR is approved
```
Comment (abroddrick) [inline docs/product/product_strategy.md:48]: I will add a link here once that PR's comments are resolved and it's merged

**Review by neilmb — COMMENTED** [review]
LGTM

**Review by PaulKuykendall — APPROVED** [review]
This looks great, Alysia.

File: docs/product/product_strategy.md [hunk 2]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+
+**Critical**- should only be determined by the product owner and means the fix for this critical bug needs to have a quick fix for it created ASAP. This is the only case where a bug fix can be added outside of the normal release cycle and directly onto the stable release.
+**High**- Can be determined by product owner or other team member, and indicates this bug is critical enough to warrant being added into the current sprint.
+**Medium**- Should be added to a sprint coming up but is not blocking users, or enough users to warrant rushing it into a sprint
+**Low**- A minor bug, that could even wait until after the next big launch date to be implemented.
+
+### Steps for Triaging
+
+1. When a bug is found, whether by a developer/designer or from feedback from an end user, a ticket should be made immediately. The actual maker of the ticket can be a member of the product team as needed.
+2. This bug ticket immediately gets a priority added Critical/High/Medium/Low, with Critcal requiring the product owner's request.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:38]: * typo at `Critcal`
* consider changing "product owner's request" to "product owner's consent"

File: docs/product/product_strategy.md [hunk 3]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+3. Anything marked as `critical` should be refined immediately and engineering should be notified via Slack that a Critical ticket has been created (if not already notified)
+4. All items not marked as `critical` by the product owner, can wait until refinement to be refined and may have their prioirty level changed during that meeting.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:40]: > by the product owner, 

Drop the comma

File: docs/product/product_strategy.md [hunk 4]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+**High**- Can be determined by product owner or other team member, and indicates this bug is critical enough to warrant being added into the current sprint.
+**Medium**- Should be added to a sprint coming up but is not blocking users, or enough users to warrant rushing it into a sprint
+**Low**- A minor bug, that could even wait until after the next big launch date to be implemented.
+
+### Steps for Triaging
+
+1. When a bug is found, whether by a developer/designer or from feedback from an end user, a ticket should be made immediately. The actual maker of the ticket can be a member of the product team as needed.
+2. This bug ticket immediately gets a priority added Critical/High/Medium/Low, with Critcal requiring the product owner's request.
+3. Anything marked as `critical` should be refined immediately and engineering should be notified via Slack that a Critical ticket has been created (if not already notified)
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:39]: > notified via Slack 

notified in our Slack's dev channel

File: docs/product/product_strategy.md [hunk 5]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+4. All items not marked as `critical` by the product owner, can wait until refinement to be refined and may have their prioirty level changed during that meeting.
+
+### Steps for dealing with Critical Bugs
+
+1. Once the critical bug ticket is refined and the bug is clear, an engineer should be assigned to work on it. (No ticket, no work)
+2. At the same point, two other engineers should be assigned to review the PR once it's made. One of the reviewing engineers can be subsititued for a designer if this is a design/content/other user facing bug fix.
+3. In the case where the engineering lead is not responsive/ unavailable to assign the ticket immediatley, other engineers should see the public slack message and volunteer as needed. The product team will be backup to make sure someone is assigned or volunteers for the ticket/PR review ASAP.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:46]: > is not responsive/ unavailable 

Consider "is unresponsive or unavailable"

> immediatley

Revise to immediately, immediately

>other engineers should see the public slack message and volunteer as needed.

Consider "the product team will make sure an engineer volunteers or is assigned to the ticket/PR review ASAP."

File: docs/product/product_strategy.md [hunk 6]
```diff
@@ -22,3 +22,27 @@ CISA lacks a scalable, efficient, and secure method of managing the .gov TLD pro
...
+4. Once done the developer, now needs to make a PR as usual but should tag the assigned PR reviewers in a public slack message stating that the PR is now waiting on their review. These reviewers should drop other tasks in order to review this promptly.
```
Comment (h-m-f-t) [inline docs/product/product_strategy.md:47]: >  Once done the developer, now needs

Once done, the developer must make a PR

> in a public slack message

in our Slack's dev channel

**Review by h-m-f-t — DISMISSED** [review]
Requesting changes on a few small copy-edits. 

I think this is solid content – but I'm unsure it belongs in product_strategy.md. I think a better fit is in this repo's /docs/operations/README.md or perhaps in a new file in the /docs/operations directory.

**Comment by abroddrick** [comment]
@h-m-f-t appreciate the review all changes should now be made!

**Comment by h-m-f-t** [comment]
Thanks! Can we move this content to `/docs/operations/README.md` or create a new file the `/docs/operations` directory? I don't think it fits in the product strategy.

**Comment by abroddrick** [comment]
> Thanks! Can we move this content to `/docs/operations/README.md` or create a new file the `/docs/operations` directory? I don't think it fits in the product strategy.

Ironically I had it in that readme first, should have gone with the first instinct! It will only take a minute to move it to the operations readme, doing that now!

**Pull request merged**